### PR TITLE
Iterate for discussion.

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

File: content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md [hunk 1]
```diff
@@ -155,6 +154,24 @@ complete the following steps:
 
     <img src="https://8026b2e3760e2433679c-fffceaebb8c6ee053c935e8915a3fbe7.ssl.cf2.rackcdn.com/field/image/grafana-mysql-data.png" width="428" height="337" />
 
+#### (Optional) Use Annotation
+With annotation support, users can submit change event to show along with the graph, adding additional information for the graph on the dashboard. See [Request an annotation](https://developer.rackspace.com/docs/metrics/v2/developer-guide/#retrieve-an-annotation) for additional information.
```
Comment (wolfdancer) [inline content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md]: Would it make sense to change the link to https://github.com/rackerlabs/rackspace-how-to/pull/513/

File: content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md [hunk 2]
```diff
@@ -155,6 +154,24 @@ complete the following steps:
 
     <img src="https://8026b2e3760e2433679c-fffceaebb8c6ee053c935e8915a3fbe7.ssl.cf2.rackcdn.com/field/image/grafana-mysql-data.png" width="428" height="337" />
 
+#### (Optional) Use Annotation
+With annotation support, users can submit change event to show along with the graph, adding additional information for the graph on the dashboard. See [Request an annotation](https://developer.rackspace.com/docs/metrics/v2/developer-guide/#retrieve-an-annotation) for additional information. 
+
+#### (Optional) Use Enum metrics
+
+The enum metrics release will allow you to push state based metrics (enums) and get the report from API on the frequency of occurrence of the values. For example, a common report for the IT teams to provide the uptime report based on the occurrences of the response codes from the server, or the state of the alerts during a period of time. Enum support is a stepping stone by providing the data to be used in report or visualization.
+
```
Comment (wolfdancer) [inline content/rackspace-metrics/create-a-grafana-dashboard-for-rackspace-metrics.md]: Would it make sense to change everything below to the article in https://github.com/rackerlabs/rackspace-how-to/pull/513/?

**Comment by kylelaffoon** [comment]
@kmsholcomb Please edit at your convenience.

**Comment by kmsholcomb** [comment]
Hi @kylelaffoon: My edits are in the attached Word doc (and in the attached PDF, which is a copy of the Word doc). I had some questions about what the dashboard is supposed to show me, and how Metrics/Intelligence/Grafana are connected. If the questions are irrelevant, ignore them. I also think that the article could be improved by taking new screenshots, but that isn't completely necessary if time it tight. Please let me know if you have any questions. Thanks!

[Create a Grafana dashboard for Rackspace Metrics_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/145229/Create.a.Grafana.dashboard.for.Rackspace.Metrics_editsKH.pdf)
[Create a Grafana dashboard for Rackspace Metrics_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/145230/Create.a.Grafana.dashboard.for.Rackspace.Metrics_editsKH.docx)

**Comment by wolfdancer** [comment]
hi @kmsholcomb and @kylelaffoon : Let me take a look at the word doc and responding to Kelly's questions.  I should be able to get it today.  Thanks

**Comment by kylelaffoon** [comment]
Thanks for your help, Shane.

Here is the document we looked over.


-Kyle

**Comment by wolfdancer** [comment]
Thanks!
--
Shane Duan
Product Manager, Monitoring and Metrics
Know your infrastructure through agents

**Pull request merged**